pkg/pipeline/input/video.go

```diff
@@ -247,7 +247,7 @@ func (v *VideoInput) buildEncoder(p *config.PipelineConfig) error {
 		}
 
 		if err = caps.SetProperty("caps", gst.NewCapsFromString(
-			fmt.Sprintf("video/x-h264,profile=%s", p.VideoProfile),
+			fmt.Sprintf("video/x-h264,profile=%s,framerate=%d/1", p.VideoProfile, p.Framerate),
```

Comment:
With what kind of output? In the the track composite case, we have a a max-framerate setting on the video rate plugin to make sure we drop frames as appropriate (and adjust the caps accordingly). In the room composite case, we set the framerate on the ximagesrc. There is no video rate plugin for room composite, so it's unclear what setting the caps  on the encoder does in this case, if there is a framerate mismatch between the caps from the ximagesrc, and the ones from the encoder (failure?)

Reply:
track composite, see dotfile posted in slack

Reply:
this is setting the framerate on the encoded caps, but that framerate is being negotiated back to `videorate`. Without it, it stays at `framerate=0/1`, and for some reason ignores `max-rate`. If we want to prevent duplicating frames, we can just set `drop-only=true` on videorate